### ISSUE TITLE
Nunjucks Custom Filters not working

### ISSUE THREAD

**Issue by christiankyle-ching** [promptfoo/promptfoo#584]
Example template seem to be not working anymore: https://github.com/promptfoo/promptfoo/tree/main/examples/nunjucks-custom-filters

When viewing YAML via the `promptfoo view`, the `nunjucksFilters` is missing.

Thanks.

**Comment by typpo** [comment]
Hi @christiankyle-ching,

Can you elaborate on "not working"?  It does look like the `allcaps` filter is working as intended in the example:

![llm eval with nunjucks filter](https://github.com/promptfoo/promptfoo/assets/310310/63a1bbb4-5e8b-4a27-a3d0-7209500917f8)

I agree that it should appear in the "View YAML" modal, though.

**Comment by christiankyle-ching** [comment]
My bad. It seems to be an issue with directories.

I am using `npx` on the root directory. When the `prompfooconfig.yaml` is placed inside a subdirectory, I also moved the `allcaps.js` in the same subdirectory, to which I got the error:
```
Template render error: (unknown path)
  Error: filter not found: allcaps
```

It worked when the `allcaps.js` is in the root directory, while the config is in any subdirectory.

I assumed the `- ./allcaps.js` line assumes relative to the config file, but it seems to be relative to the location where the script was ran. Thanks.
